### ISSUE TITLE
ZAddArgs returns syntax error

### ISSUE THREAD

**Issue by fuzaylov** [redis/go-redis#2487]
I am trying to use `GT` option of `zadd`. I found a unit test in `commands_test.go` that uses `ZAddArgs` method to use the `GT` option:
```go
// Test only the GT+LT options.
added, err := client.ZAddArgs(ctx, "zset", redis.ZAddArgs{
   GT:      true,
   Members: []redis.Z{{Score: 1, Member: "one"}},
}).Result()
```

When I apply the same strategy in my code, I get an error from Redis: `syntax error` without other explanations. Am I doing something wrong? This is my code:

```go
score := 50
member := "Tester"
members := []redis.Z{{Score: score, Member: member}}
_, err := client.ZAddArgs(ctx, config.Key, redis.ZAddArgs{
	GT: true,
	Members: members,
}).Result()

if err != nil {
	panic(err.Error()) // results in "ERR syntax error"
}
```

**Comment by monkey92t** [comment]
Please provide: go versoin, redis-server version, go-redis version

**Comment by fuzaylov** [comment]
Go: 1.18
Redis: 5.0.0
go-redis: 8.11.5

According to the Redis docs, [ZADD](https://redis.io/commands/zadd/) command is available since 1.2.0

**Comment by monkey92t** [comment]
redis-doc: "Starting with Redis version 6.2.0: Added the GT and LT options." 

you set: GT: true
